doc/source/general/cpu_hardware.rst

```diff
@@ -43,6 +43,14 @@ Triggers are all of the following:
 -  ON condition { some commands }.
 -  WHEN condition THEN { some commands }.
 
+.. note::
+
+    The :ref:`WAIT <wait>` command only causes mainline code
+    to be suspended.  Trigger code such as WHEN, ON, LOCK STEERING,
+    and LOCK THROTTLE, will continue executing while your program
+    is sitting still on the WAIT command.
+
```

Comment:
NOTE: This documentation notice is NOT referring to a change I made.  This was actually true all along even before this Pull Request.  I only just *noticed* that it was true because I was looking at that code, and decided that it should be clarified in the docs.  It surprised me when I found this out and it would probably not be what others would have assumed either.

I considered making an issue to change it so it *does* freeze everything even the triggers, until I remembered that triggers are used by flybywire's settings and thought that might open a can of worms to freeze it's work too when the script does a WAIT.